src/LuaEngine/PlayerHooks.cpp

```diff
@@ -708,3 +708,11 @@ void Eluna::OnRemoveAura(Player* player, Aura* aura, bool isExpired)
     Push(isExpired);
     CallAllFunctions(PlayerEventBindings, key);
 }
+
+void Eluna::OnBattlegroundDesertion(Player* player, const BattlegroundDesertionType type)
+{
+    START_HOOK(PLAYER_EVENT_ON_BG_DESERTION);
+    Push(player);
+    Push(type);
+    CallAllFunctions(PlayerEventBindings, key);
+}
```

Comment:
Thanks for the PR. THere's a missing new line at the end of that file. See the red circle with a line inside.

Reply:
Fixed now @55Honey